src/util/profile/prof_file.c

```diff
@@ -60,6 +59,10 @@ int profile_library_initializer(void)
 
     return k5_mutex_finish_init(&g_shared_trees_mutex);
 }
+
+#ifdef _WIN32
```

Comment:
I do not know what makes it different.  Appveyor build has been fixed, but the fix is admittedly not great.